lib/server/utilities/findAndCountAllToHttpView.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+/**
+ * @typedef CountedData
+ * @property {Number} count
+ * @property {Object[]} rows
+ */
+
+/**
+ * @typedef CountedDataView
+ * @property {{totalCount: Number, pageCount: Number}} meta
+ * @property {Object[]} data
+ */
+
+/**
+ * Adapt services and usecases results of type 'findAndCountAll' to api
+ * @param {CountedData} data count from findAndCountAll method
+ * @returns {CountedDataView} adapted result
+ */
+function findAndCountAllToHttpView({ count, rows }) {
+    count = Array.isArray(count) ? count.length : count;
```

Comment:
This is really weird and in my opinion should not be done here. The count in your `CountedData` should be the count you send back through the API, it simply convert between layer and should not have some tricks like this

Reply:
yes refactored, but ... as it's not belong to scope of this PR I will add in another one